lib/promptConfig.js

```diff
@@ -29,7 +30,12 @@ const questions = (choicesList) => {
       type: 'input',
       name: 'description',
       message: 'Enter your commit message:',
-      validate: input => (input ? true : 'A commit message is mandatory!'),
+      validate: (input) => {
+        if (checkRulesEndWithDot(input) === false) return 'The commit message can not end with a dot';
+        else if (checkRulesMaxLength(input) === false) return `The commit message is not allowed to be longer as ${configuration.rules['max-char']}. Consider writing a body.`;
+        else if (checkRulesMinLength(input) === false) return `The commit message hast to be at least ${configuration.rules['min-char']} character.`;
+        return true;
```

Comment:
I just though about one small thing I forgot. Maybe the `availableRules.js` should look like following:

```js
const rules = {
  endWithDot: {
    message: 'The commit message...',
    isRule: (input) => {
      // some if statements
    }
};
```

Maybe `isRule` is not the best. But we need something when the rule should be active.